packages/nimble-components/package.json

```diff
@@ -44,7 +44,7 @@
     "test-webkit:verbose": "karma start karma.conf.verbose.js --browsers=WebkitHeadless --single-run --skip-tags SkipWebkit",
     "test-webkit:watch": "karma start karma.conf.js --browsers=WebkitHeadless --skip-tags SkipWebkit --watch-extensions js",
     "test-webkit": "karma start karma.conf.js --browsers=WebkitHeadless --single-run --skip-tags SkipWebkit",
-    "test-concurrent": "concurrently --timings --group \"npm:test-chrome:verbose\" \"npm:test-firefox:verbose\""
+    "test-concurrent": "concurrently --timings --group \"npm:test-chrome:verbose\" \"npm:test-firefox:verbose\" \"npm:test-webkit:verbose\""
```

Comment:
Only request since we don't do it on CI.
@m-akinc can you sanity check this running locally on windows?
@jattasNI can you sanity check this running locally on mac?
If all good then lets go!

Reply:
Works on windows

Reply:
Sorry I didn't get a chance to test this before vacation. I'll prioritize it next week.

Reply:
Goal of asking for the mac audit was primarily for Jesse's benefit as our main mac dev. So I think we can go ahead and merge and Jesse can try it out when has a chance.

Reply:
Confirmed it is ok on mac. There's a known (to me) issue where many/all of the `#SkipWebkit` tests fail in Firefox but that's pre-existing and I haven't prioritized finding a good way to disable them just on macOS.